lib/Models/CsvCatalogItem.js

```diff
@@ -274,6 +275,19 @@ defineProperties(CsvCatalogItem.prototype, {
         get : function() {
             return this._tableDataSource;
         }
+    },
+
+    /**
+     * Gets the concepts that can be used to filter this data, such as Region Type, Age, etc.
+     * @type {AbsConcept[]}
```

Comment:
I guess it's a bit late to comment on this now - but just noticed this jsdoc ref isn't right.